### PR TITLE
feat!: disable slashing

### PR DESCRIPTION
**Pull request by martinfridrich**
## Description
Delay slashing to infinity so slashes can be canceled with sudo

## Related Issue
#145

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -456,6 +459,12 @@ parameter_types! {
 	pub MinSolutionScoreBump: Perbill = Perbill::from_rational_approximation(5u32, 10_000);
 }
 
+type SlashCancelOrigin = EnsureOneOf<
+	AccountId,
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<_3, _4, AccountId, CouncilCollective>,
```
Comment (green-jay) [inline runtime/src/lib.rs:465]: as there will be only once council member, EnsureMember makes a bit more sense

**Review by green-jay — COMMENTED** [review]
just ooc, is the equivocation fix somehow related to slashing?

File: runtime/src/lib.rs (hunk 1 above)
Comment (martinfridrich) [reply runtime/src/lib.rs:465]: I have to fix babe/grandpa equivocations at first to slashing start to works. I can make it in 2 issues/PRs if we want

Comment (green-jay) [reply runtime/src/lib.rs:465]: Ok so the slashing wasn't working in the first place, got it. Probably no need to split it at this point.

**Comment by jak-pan** [comment]
Bump runtime and spec_version pls

**Comment by jak-pan** [comment]
Oops. I'll fix this.

**Review by jak-pan — APPROVED** [review]

**Pull request merged**